common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp

```diff
@@ -76,11 +76,9 @@ QImage ScopedPixelBuffer::getQImage(OverlayObject & overlay, QColor & bg_color)
   return getQImage(overlay.getTextureWidth(), overlay.getTextureHeight(), bg_color);
 }
 
-OverlayObject::OverlayObject(
-  Ogre::SceneManager * manager, rclcpp::Logger logger, const std::string & name)
-: name_(name), logger_(logger)
+OverlayObject::OverlayObject(const std::string & name)
+: name_(name)
 {
-  rviz_rendering::RenderSystem::get()->prepareOverlays(manager);
```

Comment:
[Question] Why `manager` dependency was removed? Is that necessary or something like refactoring?

Reply:
I thought this line was necessary for overlay visualization when porting this plugin, but I think it's just my misunderstanding or some rviz updates make it unnecessary.